src/main/java/ch/njol/skript/aliases/ItemType.java

```diff
@@ -37,10 +19,7 @@
 import ch.njol.yggdrasil.Fields;
 import ch.njol.yggdrasil.Fields.FieldContext;
 import ch.njol.yggdrasil.YggdrasilSerializable.YggdrasilExtendedSerializable;
-import org.bukkit.Location;
-import org.bukkit.Material;
-import org.bukkit.OfflinePlayer;
-import org.bukkit.Tag;
+import org.bukkit.*;
```

Comment:
This is an insane wild card import. While you still have other bukkit imports? What is your max limit of common denominator imports setting in your IDE before it considers wild card imports? Most default values are 99.

Reply:
Good point I'll change that. I think 5 imports in most cases; sometimes even with single import (e.g. java.awt package) whcih is weird. But I haven't touched that setting in IntelliJ so I suppose I have it on the default value.